test/test_views.py

```diff
@@ -31,11 +31,12 @@ def _create_storage(self):
     def _create_blogging_engine(self):
         return BloggingEngine(self.app, self.storage)
 
-    def setUp(self):
+    def setUp(self, **config):
```

Comment:
The UnitTest standard signature for `setUp` seems to be be just `setUp(self)` per [unittest docs](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.setUp).  One workaround is to implement `other_config()` method which will just return an empty dict in the parent class but can be overridden in `TestViewsWithBootstrap` to inject the required parameters.

More than anything, I am concerned if this could have the wrong `setUp` to be called down the line.